DTXManiaプロジェクト/コード/ステージ/05.選曲/CActSelect難易度選択画面.cs

```diff
@@ -180,7 +180,7 @@ public override void OnManagedリソースの作成()
             this.tx説明背景 = CDTXMania.tテクスチャの生成( CSkin.Path( @"Graphics\5_information_BG.png" ) );
             this.tx説明1 = CDTXMania.tテクスチャの生成( CSkin.Path( @"Graphics\5_information.png" ) );
 
-            this.soundSelectAnnounce = CDTXMania.Sound管理.tサウンドを生成する( CSkin.Path( @"Sounds\DiffSelect.ogg" ) );
+            this.soundSelectAnnounce = CDTXMania.Sound管理.tサウンドを生成する( CSkin.Path( @"Sounds\DiffSelect.ogg" ), ESoundGroup.SoundEffect );
```

Comment:
Here we encounter our first example of the new ESoundGroup enumeration, used to specify the group for a given CSound to be placed with for ease of controlling the playback levels of the entire group.